site/content/in-dev/unreleased/getting-started/minio.md

```diff
@@ -0,0 +1,114 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+Title: Using MinIO
+type: docs
+weight: 500
+---
+
+In this guide we walk through setting up a simple Polaris Server with local [MinIO](https://www.min.io/) storage.
+
+Similar configurations are expected to work with other S3-compatible systems that also have the
+[STS](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) API.
+
+# Setup
+
+Start MinIO. Here's a `docker` example.
+
+```shell
+docker run -p 9100:9000 -p 9101:9001 --name minio \
+ -e "MINIO_ROOT_USER=miniouser" -e "MINIO_ROOT_PASSWORD=miniopwd" \
+ quay.io/minio/minio:latest server /data --console-address ":9001"
```

Comment:
In general the formatting of the code blocks in this doc is different than the other docs

Reply:
What do you mean by "different"? Could you give a specific pointer?

Reply:
It's gone now, but the lines were being wrapped quite short and at awkward locations. All good now!